docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -84,14 +84,17 @@ SQL support
 -----------
 
 The connector provides **read access and limited write access** to data and
-metadata in Iceberg. The following section lists all supported SQL statements
-and provides further details.
+metadata in Iceberg. The following section lists supported SQL statements and
+provides further details.
 
-  .. important::
+.. important::
 
-      No other SQL statements are supported.
+    The connector supports the following SQL statements in addition to the
+    :ref:`globally available statements <sql-globally-available>`. No other SQL
+    statements are supported.
 
-Read access:
+
+Read operations:
```

Comment:
Remove, these are always available

Reply:
all of them? For all connectors? And then also not mention them in the connectors each but instead say that all read operations as also supported?

Reply:
We can have a one sentence description (max) in the boilerplate section for the connector and discuss it more on the linked overview page. We don't need to list every read operation on the page for every connector.

Reply:
I think I got a nice solution now